Microsoft.Toolkit.Uwp.UI.Controls/ImageCropper/ImageCropper.Constants.cs

```diff
@@ -0,0 +1,72 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// The <see cref="ImageCropper"/> control allows user to crop image freely.
+    /// </summary>
+    public partial class ImageCropper
+    {
+        /// <summary>
+        /// Key of the root layout container.
+        /// </summary>
+        private const string LayoutGridName = "PART_LayoutGrid";
```

Comment:
Add ` [TemplatePart]` Attribute in ImageCropper.

         [TemplatePart(Name = LayoutGridName, Type = typeof(Grid))]
         [TemplatePart(Name = ImageCanvasPartName, Type = typeof(xx))]